src/scenes/Projects/Members/List/ProjectMembersList.tsx

```diff
@@ -50,6 +52,18 @@ export const ProjectMembersList: FC = () => {
     createProjectMemberDialogState,
     openCreateProjectMemberDialog,
   ] = useDialog();
+
+  const [
+    updateProjectMemberDialogState,
+    openUpdateProjectMemberDialog,
+    projectMemberProps,
+  ] = useDialog<{
+    projectMemberId?: string;
+    userId?: string;
+    userRoles?: readonly Role[];
+    projectId: string;
+  }>();
```

Comment:
This fields should just all be required, to match `UpdateProjectMemberProps`. Then we don't have to mess with passing empty strings if they're undefined.

Reply:
set to use `UpdateProjectMemberProps` as suggested good idea